doc/lsst.source.injection/reference/41_match_injected_outputs.rst

```diff
@@ -0,0 +1,201 @@
+.. _lsst.source.injection-ref-inspect:
+
+========================
+ Match Injected Outputs
+========================
+
+-----------------------
+ Process injected data
+-----------------------
+To create an injected outputs table,
+In ``drp_pipe`` there is a standard data processing pipeline for coadd-level injection.
+This pipeline is dynamically updated using ``make_injection_pipeline``.
+This example ``pipetask run`` command makes use of the ``injected_step3`` pipeline subset to run only tasks including and after ``inject_coadd`` within the typical ``step3``.
+This saves memory and runtime by skipping the expensive ``makeWarp`` and ``assembleCoadd`` tasks which usually occur before ``inject_coadd`` in the default ``step3``.
+
+.. code::
+
+    pipetask --long-log --log-file $LOGFILE \
+    run --register-dataset-types \
+    -b $REPO \
+    -i $INPUT_DATA_COLL,$INJECTION_CATALOG_COLL \
+    -o $PROCESSED_DATA_COLL \
+    -p $DRP_PIPE_DIR/pipelines/HSC/DRP-RC2_subset+injected_deepCoadd.yaml#injected_step3 \
+    -d "instrument='HSC' AND skymap='hsc_rings_v1' AND tract=9813 AND patch=42 AND band='i'"
+
+*where*
+
+    `$LOGFILE`
+        The full path to a user-defined output log file.
+
+    `$REPO`
+        The path to the butler repository.
+
+    `$INPUT_DATA_COLL`
+        The name of the input data collection.
+
+    `$INJECTION_CATALOG_COLL`
+        The name of the input injection catalog collection.
+
+    `$PROCESSED_DATA_COLL`
+        The name of the injected output collection.
+
+.. note::
+    While this example makes use of ``injected_step3`` within a coadd-level injection pipeline, it is also possible to run ``injected_step1`` within an exposure- or visit-level pipeline.
+    Regardless of injection type, you can then elect to continue processing by running any following ``injected_`` steps.
```

Comment:
What's the purpose of this section, relative to prior sections in this quick reference guide series? If the users are clicking through, by this point in time they should have:
* made an injection pipeline YAML
* made an injection catalog
* injected their sources
* visualized their outputs

Having chatted with you yesterday, I think this can be moved into the injection section perhaps? And, if you don't want to repeat all the `pipetask run` info there, you can mention the `injected_stepN` logic in an info box instead perhaps?